### DISCUSSION TITLE
How to query by linked records via REST API?

### DISCUSSION THREAD

**Discussion by RawmanGrey** [nocodb/nocodb#1441] [Q&A] [answered]
Hello, community! 🙏 Thank you for the great project! 

I've been struggling with a necessity to filter REST API call result by target relation property. GET API calls on tables with relations return `table_nameRead` keys with (tunable via mm, hm and bt) property objects, but using `table_nameRead` in [where=](https://docs.nocodb.com/developer-resources/rest-apis/#comparison-operators) parameter returns fair error that this column doesn't exist. Same error met when querying for real relation column's name. Am I missing something and filtering by relations' columns is possible?

It seems weird that Lookup columns are not returned in API calls, though Rollups are, - that could be a solution for my problem. I guess, that implementing #512 feature is relative and will help to enable Lookup columns in API calls results.

**Comment by o1lab** [comment, marked as answer]
Filtering via target relation is coming soon. Stay tuned.

**Reply by marcomow** [reply]
Hi, I couldn't find any documentation about this, is the feature implemented?

**Reply by pashkatrick** [reply]
Any updates here?

**Reply by snoseeds** [reply]
Any updates?

**Reply by SkyLeite** [reply]
I guess not
